src/components/Navbar/index.view.tsx

```diff
@@ -8,14 +16,96 @@ interface NavbarProps {
 }
 
 const Navbar: React.FC<NavbarProps> = ({ title, buttonProps }: NavbarProps) => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const useStyles = makeStyles({
+    buttonWrapper: {
+      margin: "0 calc(5% + var(--small-spacer)) 0 auto",
```

Comment:
The margin here isn't responsive enough to the viewport width at tablet breakpoints; there should be a bit more space between the banner & hamburger button, especially since both are clickable links
![Screenshot from 2021-01-02 17-21-10](https://user-images.githubusercontent.com/19658882/103470053-03a05c80-4d22-11eb-812a-77ebf7bd8bc6.png)